src/EditorFeatures/CSharpTest/ReplaceConditionalWithStatements/ReplaceConditionalWithStatementsTests.cs

```diff
@@ -0,0 +1,1166 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.CSharp.ReplaceConditionalWithStatements;
+using Microsoft.CodeAnalysis.Editor.UnitTests.CodeActions;
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.Editor.CSharp.UnitTests.ReplaceConditionalWithStatements;
+
+using VerifyCS = CSharpCodeRefactoringVerifier<CSharpReplaceConditionalWithStatementsCodeRefactoringProvider>;
+
+public class ReplaceConditionalWithStatementsTests
```

Comment:
Are nested ternary operations out of the scope of this PR?

Reply:
Nope!  they would be supported.  you should be able to invoke on the outer or inner.  I can add test for that :)

Reply:
tests added.